### ISSUE TITLE
App crash on Stop Sound in IOS 

### ISSUE THREAD

**Issue by Abosrie** [ukasz123/soundpool#37]
I am trying to stop the sound by calling the below function but the app is crashing

```
Future<void> _stopSound() async {
    if (ringing != null) {
      await _soundpool.stop(ringing);
    }
  }
```

**Comment by ukasz123** [comment]
Can you give me more information about the crash? What plugin version are you using? Can you share the error log?

**Comment by Abosrie** [comment]
@ukasz123 - I am using ->soundpool: ^1.0.1

There is no error log the simulator is just getting disconnected.

**Comment by ukasz123** [comment]
@Abosrie I encourage you to upgrade to 1.1.0 - I remember fixing some issues happening on iOS. Also it includes some extra assertions that may help in debugging.

I guess you need to build an iOS app from Xcode instead of using `flutter run`.Xcode should keep connection even after app crashes.

**Comment by Abosrie** [comment]
I seem that the Stop() method is not working even in the example in the package! the only way to stop the sound is using _soundpool.release();

**Comment by ukasz123** [comment]
Should be fixed in version `1.1.1`.
